### ISSUE TITLE
v6.6.0 🙏

### ISSUE THREAD

**Issue by msrumon** [kothing/fontawesome-pro#1]

**Comment by dearvoodoo** [comment]
[╰(*°▽°*)╯](https://file.io/Gs1tM9kUfDpV)

**Comment by dearvoodoo** [comment]
V.6.7.2 -> [╰(*°▽°*)╯](https://mega.nz/file/RioXCThC#DwKOCPBX--8sLDJvzkquGqF-0G1duFuGeMHoDkS3tn0) 
I'm hiding the link to prevent the DMCA from removing it too quickly.

**Comment by tomgie** [comment]
Can you upload 7.0.0 please?

**Comment by dearvoodoo** [comment]
I'll upload 7.0.0 as soon as I can get my hands on it.

**Comment by dearvoodoo** [comment]
V.7.0.0 Pro Web -> [╰(*°▽°*)╯](https://mega.nz/file/U6pgCDSR#S5P6aHsksl5CuUPkk23dSh9UMRSXJOxNcYVmgtR52o0) 
I'm hiding the link to prevent the DMCA from removing it too quickly.

For the moment, this is only the web version. I’ll share the complete version as usual when I get it. I’m also searching for the Pro+ version.

**Comment by DogancanYr** [comment]
> So, like, you guys definitely know that you are stealing this but are cool with it?

Exactly. Besides, how could someone developing something on their own pay $750 a year? We're not a corporation.
